doc/releases/changelog-dev.md

```diff
@@ -299,6 +299,15 @@
   
 **Operator Arithmetic:**
 
+* `default.qubit` now will natively execute any operation that defines a matrix except
+  for trainable `Pow` operations. This includes custom operations, `GroverOperator`, `QFT`,
+  `U1`, `U2`, `U3`, and arithmetic operations. The existance of a matrix is determined by the
+  `Operator.has_matrix` property.
+
+  When adjoint differentiation is requested, tapes are now pre-expanded in `qml.execute` so
+  that all trainable operations have a generator.
+  [(#2836)](https://github.com/PennyLaneAI/pennylane/pull/2836)
```

Comment:
```suggestion
* When adjoint differentiation is requested, circuits are now decomposed so
  that all trainable operations have a generator.
  [(#2836)](https://github.com/PennyLaneAI/pennylane/pull/2836)
```

Reply:
I would add a separate bullet point here

Reply:
@albi3ro could you clarify here how this differed from the previous decomposition logic?

Reply:
Previously, it just decomposed `Rot` or raised an error.